splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -3074,7 +3075,7 @@
         },
         {
           "type": "string",
-          "pattern": "^(?:-1|\\d+(?:\\.\\d+)?)$"
+          "pattern": "^(?:-1|\\d+(?:\\.\\d+)?|(((\\d+,)+\\d+|(\\d+[/-]\\d+)|\\d+|\\*(/\\d*)?) ?){5})$"
```

Comment:
nit: Isn't "?:-1" part covered by first element from anyOf (const -1)?

Reply:
Looks like it's not - removing this part breaks UTs with validation. I will take a look if it can be fixed, but not under this PR